### DISCUSSION TITLE
mqtt rain rate / 10

### DISCUSSION THREAD

**Discussion by msman0** [poblabs/weewx-belchertown#862] [Q&A]
Hi, if I open the page the rain rate is ok. After the mqtt update the value is divided by 10.

For example:
rain rate: 2,0 mm
mqtt update:
rain rate: 0,2 mm

Where can I change it?
Thank you!
Regards.

**Comment by James-76** [comment]
I would take a look at bug report #510 which I think will solve your issue

James

**Comment by msman0** [comment]
Thank you!
